packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsRichTextEditBox.tsx

```diff
@@ -50,8 +58,8 @@ export const ChatMessageComponentAsRichTextEditBox = (
   const { onCancel, onSubmit, strings, message } = props;
 
   const [textValue, setTextValue] = useState<string>(message.content || '');
-  /* @conditional-compile-remove(attachment-upload) */
-  const [attachmentMetadata, _] = React.useState(getMessageWithAttachmentMetadata(message));
+  /* @conditional-compile-remove(attachment-download) @conditional-compile-remove(attachment-upload) */
+  const [attachmentMetadata, setAttachmentMetadata] = React.useState(getMessageWithAttachmentMetadata(message));
```

Comment:
Nit: We have imported useState so here can be just `const [attachmentMetadata, setAttachmentMetadata] = useState(getMessageWithAttachmentMetadata(message));`